index.js

```diff
@@ -9,15 +9,34 @@ import esbuild from 'esbuild';
 
 const ssrFunctionRoute = '/api/__render';
 
+/**
+ * Validate the static web app configuration does not override the minimum config for the adapter to work correctly.
+ * @param config {import('./types/swa').ExtendStaticWebAppConfig} 
+ * */
+function validateExtendConfig(config) {
+	if (config) {
+		if ('navigationFallback' in config) {
+			throw new Error("Cannot override navigationFallback since is used for SSR and endpoints.");
```

Comment:
Single quotes for consistency.

Reply:
Good catch. Wonder why prettier didn't kick in.